packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.tsx

```diff
@@ -79,9 +81,7 @@ const ContractDrawer = observer(
                 duration_unit={getDurationUnitText(getDurationPeriod(contract_info)) ?? ''}
                 duration={getDurationTime(contract_info)}
                 exit_spot={exit_spot}
-                has_result={
-                    !!is_sold || is_multiplier || is_vanilla || is_turbos || is_accumulator || is_smarttrader_contract
-                }
+                has_result={!!is_sold || is_multiplier || is_options || is_accumulator || is_smarttrader_contract}
```

Comment:
I'm thinking of removing this prop because we show the details for all the contracts. But im not sure if we gonna hide it from some contract in the future. Need you guys opinion

Reply:
since the point of the task to make it visible for all contracts, it's a great idea to remove the unnecessary prop and condition 👍🏻